index.html

```diff
@@ -37,15 +37,208 @@
     </header>
 
     <!-- hero content section -->
-    <main id="hero">
-      <h1 class="dev-name">Devendra Mulewa</h1>
+    <main id="hero" class="flex popins">
+      <h1 class="title playfair">Devendra Mulewa</h1>
       <p class="dev-intro">
         Hello! I am a software developer! I can help you build a product,
         feature or website. Take a look at my work. <br /><br />
         If you like what you see and have a project you need coded, don’t
         hesitate to contact me.
       </p>
-      <button id="start-collab">start collabration</button>
+      <button class="g-btn">start collabration</button>
+      <div class="hero-end">
+        <img
+          src="assets/images/hero-end-final.svg"
+          alt="hero section background"
+        />
```

Comment:
- [ ] Kindly fix the image to look exact like the original figma design according to the requirement kindly stick to the design as much as possible this will make your website look professional (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] your design
![you](https://user-images.githubusercontent.com/92458236/236251213-5f0afa2e-9af8-4ff7-9f6c-a6ac043960b0.jpg)

- [ ] figma design
![template 5](https://user-images.githubusercontent.com/92458236/236251342-17044b9d-f635-45fc-a4b2-473734e297d1.jpg)

Reply:
To download the image from figma is not that hard but I think you need help if you are available now can I call you on zoom to fix this ?but if you have already fix this that cool and great job

Reply:
Problem solved! Just making second code review request 😇